code/script.js

```diff
@@ -6,14 +6,143 @@ alert(
   `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
 )
 
+const userName = prompt("What is you name?");
+
+alert(
+  `Nice to meet you ${userName}!`
```

Comment:
I think it's really nice that you've used the username in many of the alerts/prompts since it makes it more personalised.